classes/03class/exercises/c03-docker01/gpavelar/README.md

```diff
@@ -0,0 +1,31 @@
+# C03-Docker01
+
+## Docker
+
+- [Dockerfile](Dockerfile)
+
+## Command Execution Output
+
+- Command to build the image:
+
+```bash
+## Docker build command
+docker build -t da-terraform:c03-docker-e01 .
+```
+
+- Command to run the image and its output.:
+
+```bash
+## Docker run
+docker run -it da-terraform:c03-docker-e01
+
+# Output
+Terraform v0.12.20
+
+Your version of Terraform is out of date! The latest version
+is 0.12.28. You can update by downloading from https://www.terraform.io/downloads.html
+```
+
```

Comment:
whats is the size of the image?
can you run a `docker images` and add the output?

Reply:
Yeah, I've added this output. 
``bash
da-terraform          c03-docker-e01      718a82c47a37        25 hours ago        57.8MB
```
I just need to pay attention to this one, right?